File: app/src/main/java/com/msbte/modelanswerpaper/models/CommonItemModel.java
package com.msbte.modelanswerpaper.models;

public class CommonItemModel {
    public String name = "", path = "", sub_domain = "";

    public CommonItemModel() {
    }

    public CommonItemModel(String name, String path, String sub_domain) {
        this.name = name;
        this.path = path;
        this.sub_domain = sub_domain;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public String getsubDomain() {
        return sub_domain;
    }

    public void setsubDomain(String subDomain) {
        this.sub_domain = subDomain;
    }
}

File: app/src/main/java/com/msbte/modelanswerpaper/models/CommonItemModel.kt
package com.msbte.modelanswerpaper.models

class CommonItemModel {
    var name = ""
    var path = ""
    var sub_domain = ""

    constructor() {}
    constructor(name: String, path: String, sub_domain: String) {
        this.name = name
        this.path = path
        this.sub_domain = sub_domain
    }

    fun getsubDomain(): String {
        return sub_domain
    }

    fun setsubDomain(subDomain: String) {
        sub_domain = subDomain
    }
}